source/backend/routes/users.js

```diff
@@ -35,6 +35,25 @@ router.get('/savedRecipes', async (req, res) => {
   }
 });
 
+router.get('challenge/:challenge', async (req, res) => {
+  try {
+    const { userId } = req.userInfo;
+    const { challenge } = req.params;
+
+    const completedChallenges = await completedRecipesModel.getCompletedChallenges(userId, challenge);
+
+    return res.status(200).json(completedChallenges);
+
+  } catch (err) {
+    console.error(err);
+    return res.status(503).json({
+      message: 'Failed to get user information due to'
+        + 'internal server error',
+      err,
+    });
+  }
+});
+
```

Comment:
The request endpoint is up to us. I believe when we do something like `http://localhost:3000/users/challenge/Spicy+Sips` we need to consider how it will be formatted. I guess this is something we need a bit more clarification on, but for now I think we can assume that every space will be separated by a `+` since that's generally the convention. 

TL;DR
req.params -> Spicy+Sips
it's also possible we get an invalid challenge, so add error handling to verify the challenge exists. 
Psuedocode: 
challenge = req.challenge
parse challenge -> 'Spicy+Sips' -> 'Spicy Sips'
if that's not a valid challenge: return status 400 (bad request)
otherwise: return the recipes from that challenge

Reply:
Done. Let me know if there are any issues.